### PR TITLE
chore: rename Context to EventContext

### PR DESCRIPTION
**Pull request by geyslan**
### 1. Explain what the PR does

00ffa33 **chore: rename Context to EventContext**

```
It aligns with the name of the type in the ebpf land, `event_context`
avoiding confusion with the `context.Context` type.
It also updates the respective comments.
```

### CODE REVIEW

File: pkg/bufferdecoder/protocol.go [hunk 1]
```diff
@@ -16,14 +16,15 @@ const (
...
-// it is used to unmarshal binary data and therefore should match (bit by bit) to the `context_t` struct in the ebpf code.
-// NOTE: Integers want to be aligned in memory, so if changing the format of this struct
-// keep the 1-byte 'Argnum' as the final parameter before the padding (if padding is needed).
-type Context struct {
-	Ts              uint64
+// EventContext contains common metadata that is collected for all types of events.
+//
+// NOTE:
+// - Use pahole to ensure this struct reflects the `event_context“ struct in the eBPF code.
+// - Update the `EbpfDecoder` if any change to this struct is done.
```
Comment (rafaeldtinoco) [inline pkg/bufferdecoder/protocol.go:23]: The all CAPITAL WORDS for the "PLEASE NOTE" should remain. Its a very important statement made in other places as well (I remember reviewing that PR and it was a point of discussion, so lets keep maximuma attention by keepig A HUGE COMMENT SAYING THAT, please.

**Review by rafaeldtinoco — APPROVED** [review]
LGTM but change back the comment as I said in the code, please.

File: pkg/bufferdecoder/protocol.go (hunk 1 above)
Comment (geyslan) [reply pkg/bufferdecoder/protocol.go:23]: OK, I will convert them back to CAPITAL.

**Comment by geyslan** [comment]
I made a few more renames focused on avoiding confusion with context.Context.

**Pull request merged**